### ISSUE TITLE
Finding download URL

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#19]
## User story
1. As a user after typing in the vendor's URI
2. I want the software to search for the download link
3. So that the firmware can be downloaded

## Acceptance criteria
* Message of correct execution after having correctly downloaded the firmware
* Error message in case the link isn't found

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
